frontend/my-adopthive-app/src/SignIn/SignIn.css

```diff
@@ -3,21 +3,23 @@
     justify-content: center;
     align-items: center;
     height: 100vh;
+
 }
 
 .sign-in {
-    background-color: #ccc;
     padding: 70px;
     border-radius: 5px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
     text-align: center;
     max-width: 400px;
     width: 100%;
+    /* background-color: #ccc;
```

Comment:
Remove commented out code again. This applies to all instances so I'll trust you to remove them by the next version